lib/alertStateMachine.js

```diff
@@ -1,9 +1,10 @@
 const CHATBOT_STATE = require('./chatbotStateEnum')
 
 class AlertStateMachine {
-  constructor(getReturnMessageToRespondedByPhoneNumber, getReturnMessageToOtherResponderPhonesNumbers) {
+  constructor(getReturnMessageToRespondedByPhoneNumber, getReturnMessageToOtherResponderPhonesNumbers, getExpectedRequestToResetMessage) {
```

Comment:
Rename "getExpectedRequestToResetMessage" to "getClientMessageForRequestToReset"; I think that is more transparent.